data/data/aws/vpc/vpc.tf

```diff
@@ -18,3 +18,15 @@ resource "aws_vpc_endpoint" "s3" {
   service_name    = "com.amazonaws.${var.region}.s3"
   route_table_ids = ["${concat(aws_route_table.private_routes.*.id, aws_route_table.default.*.id)}"]
 }
+
+resource "aws_vpc_dhcp_options" "main" {
+  domain_name         = "${var.region == "us-east-1" ? "ec2.internal" : format("%s.compute.internal", var.region)}"
```

Comment:
I don't understand this check? 
Does us-east-1 have a different internal domain from other regions?

Reply:
NM, https://docs.aws.amazon.com/vpc/latest/userguide/VPC_DHCP_Options.html#DHCPOptionSets explains this: 

> If you're using AmazonProvidedDNS in us-east-1, specify ec2.internal. If you're using AmazonProvidedDNS in another region, specify region.compute.internal (for example, ap-northeast-1.compute.internal).

Reply:
> NM, https://docs.aws.amazon.com/vpc/latest/userguide/VPC_DHCP_Options.html#DHCPOptionSets explains this...

And I quote that section in the d3f3579a4 commit message and PR topic post ;).